doc/guides/kconfig/index.rst

```diff
@@ -816,21 +816,27 @@ Devicetree Related Functions
 ============================
 
 See the Python docstrings in ``scripts/kconfig/kconfigfunctions.py`` for more
-details on the functions.
+details on the functions.  The `*_int` version of each function return the
```

Comment:
Return/returns typo (though I had that in the original version too).

Reply:
fixed